assets/js/algolia-search.js

```diff
@@ -0,0 +1,102 @@
+const searchClient = algoliasearch(
+  "1V7DZGZJKK",
+  "0ba2c5f100ff5fd004415e4abbcf9b9c"
+);
+
+const search = instantsearch({
+  indexName: "ogp_egazettes_index",
+  searchClient,
+});
+
+// Note: Publish date is formatted as YYYY-MM-DD
+search.addWidgets([
+  instantsearch.widgets.searchBox({
+    container: "#searchbox",
+    autofocus: true,
+  }),
+  instantsearch.widgets.hits({
+    container: "#hits",
+    templates: {
+      item(hit) {
+        return `
+            <h5 class="search-results">
+            <a class="search-content" href=${
+              hit.fileUrl
+            } type="application/pdf">${instantsearch.highlight({
+          attribute: "title",
+          highlightedTagName: "mark",
+          hit,
+        })}</a>
+            <p class="search-content permalink">${hit.fileUrl}</p>
+            <p class="search-content permalink">Category: ${instantsearch.highlight(
```

Comment:
nit: I think we should use a different CSS class for this instead of `permalink`? Same for the rest that are not permalinks.

Reply:
changed in e5f300b!